### PR TITLE
Add basic keyboard navigation

### PR DESCRIPTION
**Pull request by dgliwka**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -83,6 +85,67 @@ export default class App extends Component<{
...
+        {
+            const child = node.children[0]
+        
+            this.setState({
+                chosenNode: child.id
+            })
+        }
+    }
+
+    selectNextChildNode = () =>
```
Comment (dgliwka) [inline src/App.tsx:115]: Can I somehow simplify these two methods?

File: src/core/keyMap.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const keyMap = {
+    LEFT: ["h", "left"],
+    RIGHT: ["l", "right"],
+    UP: ["k", "up"],
+    DOWN: ["j", "down"],
+    DELETE_NODE: ["del", "d"],
+};
+
+export default keyMap;
```
Comment (allgreed) [inline src/core/keyMap.ts:9]: nit: newnline much? ^^

File: src/core/keyMap.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const keyMap = {
```
Comment (allgreed) [inline src/core/keyMap.ts:1]: I don't think this belongs to the core logic, IMO it's more of an UI thing. Does that make sense?

File: src/App.tsx (hunk 1 above)
Comment (allgreed) [reply src/App.tsx:115]: Got a few ideas, but let's extract the component first

**Pull request merged**